src/include/parser/parsenodes.h

```diff
@@ -228,172 +222,182 @@ typedef struct SelectStmt {
-  bool		if_not_exists;	/* just do nothing if index already exists? */
+typedef struct IndexStmt {
+  NodeTag type;
+  char *idxname;        /* name of new index, or NULL for default */
+  RangeVar *relation;   /* relation to build index on */
+  char *accessMethod;   /* name of access method (eg. btree) */
+  char *tableSpace;     /* tablespace, or NULL for default */
+  List *indexParams;    /* columns to index: a list of IndexElem */
+  List *options;        /* WITH clause options: a list of DefElem */
+  Node *whereClause;    /* qualification (partial-index predicate) */
+  List *excludeOpNames; /* exclusion operator names, or NIL if none */
+  char *idxcomment;     /* comment to apply to index, or NULL */
+  Oid indexOid;         /* OID of an existing index, if any */
+  Oid oldNode;          /* relfilenode of existing storage, if any */
+  bool unique;          /* is index unique? */
+  bool primary;         /* is index a primary key? */
+  bool isconstraint;    /* is it for a pkey/unique constraint? */
+  bool deferrable;      /* is the constraint DEFERRABLE? */
+  bool initdeferred;    /* is the constraint INITIALLY DEFERRED? */
+  bool transformed;     /* true when transformIndexStmt is finished */
+  bool concurrent;      /* should this be a concurrent index build? */
+  bool if_not_exists;   /* just do nothing if index already exists? */
 } IndexStmt;
 
-typedef struct ColumnDef
-{
-NodeTag		type;
-char	   *colname;		/* name of column */
-TypeName   *typeName;		/* type of column */
-int			inhcount;		/* number of times column is inherited */
-bool		is_local;		/* column has local (non-inherited) def'n */
-bool		is_not_null;	/* NOT NULL constraint specified? */
-bool		is_from_type;	/* column definition came from table type */
-char		storage;		/* attstorage setting, or 0 for default */
-Node	   *raw_default;	/* default value (untransformed parse tree) */
-Node	   *cooked_default; /* default value (transformed expr tree) */
-Node *collClause;	/* untransformed COLLATE spec, if any */
-Oid			collOid;		/* collation OID (InvalidOid if not set) */
-List	   *constraints;	/* other constraints on column */
-List	   *fdwoptions;		/* per-column FDW options */
-int			location;		/* parse location, or -1 if none/unknown */
+typedef struct CreateTrigStmt {
+  NodeTag type;
+  char *trigname;     /* TRIGGER's name */
+  RangeVar *relation; /* relation trigger is on */
+  List *funcname;     /* qual. name of function to call */
+  List *args;         /* list of (T_String) Values or NIL */
+  bool row;           /* ROW/STATEMENT */
+  /* timing uses the TRIGGER_TYPE bits defined in catalog/pg_trigger.h */
+  int16_t timing; /* BEFORE, AFTER, or INSTEAD */
+  /* events uses the TRIGGER_TYPE bits defined in catalog/pg_trigger.h */
+  int16_t events;    /* "OR" of INSERT/UPDATE/DELETE/TRUNCATE */
+  List *columns;     /* column names, or NIL for all columns */
+  Node *whenClause;  /* qual expression, or NULL if none */
```

Comment:
Yeah, keeping the naming style consistent is really important and thanks for your suggestions. But I believe we will not change this file because it is copied from Postgres' parser without any change. Updating this part may confuse other developers in the future.